queries/toml/highlights.scm

```diff
@@ -1,8 +1,9 @@
 ; Properties
 ;-----------
 
-(bare_key) @property
+(bare_key) @type
```

Comment:
why type? Should then the section title be changed? what is the difference between `(pair (bare_key)) @property` and `bare_key`? I assume you want to change the highlight of the value, right?

Reply:
`(bare_key)` is used for both table names and property names.
A top-level `(bare_key)` is a table while `(pair (bare_key))` is a property.